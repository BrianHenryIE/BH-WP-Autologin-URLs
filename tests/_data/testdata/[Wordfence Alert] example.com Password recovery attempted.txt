This email was sent from your website "Example Site" by the Wordfence plugin at Friday 28th of June 2019 at 01:25:39 PM
The Wordfence administrative URL for this site is: http://example.org/wp-admin/admin.php?page=Wordfence
Someone tried to recover the password for user with email address: myuser@example.org
User IP: 256.256.256.256
User hostname: 256-256-256-256.static.exampleisp.com
User location: Dublin, Ireland



--
To change your alert options for Wordfence, visit:
http://example.org/wp-admin/admin.php?page=Wordfence&subpage=global_options
To see current Wordfence alerts, visit:
http://example.org/wp-admin/admin.php?page=Wordfence




No longer an administrator for this site? Click here to stop receiving security alerts: http://example.org/?_wfsf=removeAlertEmail&jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImFkbWluQGFuYWJvbGljdHYuY29tIiwiX2V4cCI6MTU2MjMzMzEzOX0.iiVqbP-3-ywdqDSSxEISL7EIYn7G5dExGDfboLqqPts