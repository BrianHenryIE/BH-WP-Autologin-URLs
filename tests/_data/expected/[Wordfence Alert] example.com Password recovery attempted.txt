This email was sent from your website "Example Site" by the Wordfence plugin at Friday 28th of June 2019 at 01:25:39 PM
The Wordfence administrative URL for this site is: http://example.org/wp-admin/admin.php?page=Wordfence&autologin=123~mockpassw0rd
Someone tried to recover the password for user with email address: myuser@example.org
User IP: 256.256.256.256
User hostname: 256-256-256-256.static.exampleisp.com
User location: Dublin, Ireland



--
To change your alert options for Wordfence, visit:
http://example.org/wp-admin/admin.php?page=Wordfence&subpage=global_options&autologin=123~mockpassw0rd
To see current Wordfence alerts, visit:
http://example.org/wp-admin/admin.php?page=Wordfence&autologin=123~mockpassw0rd




No longer an administrator for this site? Click here to stop receiving security alerts: http://example.org/?_wfsf=removeAlertEmail&jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImFkbWluQGFuYWJvbGljdHYuY29tIiwiX2V4cCI6MTU2MjMzMzEzOX0.iiVqbP-3-ywdqDSSxEISL7EIYn7G5dExGDfboLqqPts&autologin=123~mockpassw0rd